pkgs/by-name/no/nohang/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  nixosTests,
+  fetchFromGitHub,
+  python3,
+  sudo,
+  libnotify,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nohang";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "hakavlad";
+    repo = "nohang";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-aRiBGnJHdtQ5euvJ4DSuCZO5i5OJVRiCesaatrIARmg=";
+  };
+
+  postPatch = ''
+    patchShebangs src
+    substituteInPlace src/* --replace 'notify-send' '${libnotify}/bin/notify-send' --replace 'sudo' '${sudo}/bin/sudo'
```

`--replace` is deprecated - use `--replace-fail` instead (also, TIL that `libnotify`'s `mainProgram` is set to `notify-send`)
```suggestion
    substituteInPlace src/* \
      --replace-fail 'notify-send' '${lib.getExe libnotify}' \
      --replace-fail 'sudo' '${lib.getExe sudo}'
```